src/GEMItem.cpp

```diff
@@ -346,15 +398,23 @@ GEMItem::GEMItem(const char* const title_, const char* const (*getValueStr)(void
   , type(GEM_ITEM_VAL)
 { }
 
+GEMItem::GEMItem(const char* const (*getValueStr)(void))
+  : title(nullptr)
+  , getValue(getValueStr)
+  , linkedType(GEM_VAL_CALLBACK)
+  , readonly(true)
+  , type(GEM_ITEM_VAL)
+{ }
+
 GEMItem::GEMItem(const char* const title_)
   : title(title_)
   , readonly(true)
   , type(GEM_ITEM_TEXT)
 { }
 
-template<typename... Args> GEMItem::GEMItem(const __FlashStringHelper* title_, Args... args) :
-  GEMItem(reinterpret_cast<PGM_P>(title_), args...) {
-}
+// template<typename... Args> GEMItem::GEMItem(const __FlashStringHelper* title_, Args... args) :
+//   GEMItem(reinterpret_cast<PGM_P>(title_), args...) {
+// }
```

Comment:
If you do not use this any longer, shall that be removed?

Is also fine, if it would stay, cause we are not merging to the master ;)

Reply:
I would like to get it working, but for now it produces a compiler error. I will remove it